nvm.sh

```diff
@@ -3227,11 +3229,13 @@ nvm_auto() {
       nvm install >/dev/null
     fi
   elif [ "_$NVM_MODE" = '_use' ]; then
-   VERSION="$(nvm_resolve_local_alias default 2>/dev/null || nvm_echo)"
-    if [ -n "$VERSION" ]; then
-      nvm use --silent "$VERSION" >/dev/null
-    elif nvm_rc_version >/dev/null 2>&1; then
-      nvm use --silent >/dev/null
+    if [ "_$NVM_CURRENT" = '_system' ]; then
```

Comment:
The current behavior is that whenever I re-source `nvm.sh`, I get the auto-used version. With this change, that won't be the case.

(also, yes, this needs to handle "none")